includes/admin/onboarding.php

```diff
@@ -109,6 +95,11 @@ function() use ( $result ) {
 		return;
 	}
 
+	/** Add podcast author. */
```

Comment:
```suggestion
	/* Add podcast author. */
```

`/**` indicates the start of a docblock, only one asterisk is needed for the start of a comment.